### ISSUE TITLE
Load Raw Mp3 Data into AudioSegment

### ISSUE THREAD

**Issue by ComedicChimera** [jiaaro/pydub#249]
Hi.  I am trying to use **pydub** to convert and mp3 to a wav, directly from raw mp3 data.  More specifically, I am trying to do so from data read from a url using urllib.

Here is my example (i am trying to stream it):
```python
import pydub
from io import BytesIO

req = urllib.request.Request(url)
response = urllib.request.urlopen(req)
data = response.read(1024)
while data:
   sound = pydub.AudioSegment.from_mp3(BytesIO(data))
   data = response.read(1024)
```

When I attempt to run this code, I get the following error from ffmpeg:
```
Failed to read frame size: Could not seek to 2047.\r\nC:\\Users\\forlo\\AppData\\Local\\Temp\\tmpvukzjmeo: Invalid argument\r\n
```
I have tried with StringIO, but I get errors from that too.

*Note: The end goal is two use pydub to convert data to WAV and play it back as a stream through 
sounddevice. (I know it seems complicated, but I am trying to push audio through a specific output device)*

I know this a bit of niche thing, but a bit of help would be appreciated.  I have been trying to stream audio like this for nearly 6 hours, and I have yet to find a working lib.

### Metadata
Python Version: 3.6
Interpreter: Miniconda

**Comment by jiaaro** [comment]
I don’t think you can load arbitrary chunks of an mp3 file - you probably want to pipe your mp3 through ffmpeg directly (into stdin and get the converted wav data from std out)

**Comment by chess92** [comment]
you can import raw mp3 data, but it always crashes at the end

but on my mac the files play fine when exported from AudioSegment
